### ISSUE TITLE
clang++ warning -Wstring-plus-int

### ISSUE THREAD

**Issue by mslehto** [sems-server/sems#7]
Don't know how to fix following clang++ warning or if warning even is reasonable.
Some -Wstring-plus-int discussion found here https://llvm.org/bugs/show_bug.cgi?id=13541

FreeBSD 10.1
clang++ 3.4.1
```C++
clang++ -MMD -c -o jsonArg.o jsonArg.cpp -D_DEBUG -D_THREAD_SAFE -D_REENTRANT -DSEMS_VERSION='"1.6-dev-1269-ge9b03cc-dirty"' -DARCH='"x86_64"' -DOS='"freebsd"' -DSYSTEM_SAMPLECLOCK_RATE=32000LL -DSEMS_APP_NAME='"sems"' -DUSE_MONITORING -I/usr/local/include -I. -fno-strict-aliasing -DUSE_INTERNAL_RESAMPLER -Wall -Wno-reorder -Wno-unused -fPIC -g   -O2
jsonArg.cpp:97:22: warning: adding 'const char' to a string does not append to the string [-Wstring-plus-int]
          result += "\\u00" + hex_chars[*c >> 4] + hex_chars[*c & 0xf];
                    ~~~~~~~~^~~~~~~~~~~~~~~~~~~~
jsonArg.cpp:97:22: note: use array indexing to silence this warning
          result += "\\u00" + hex_chars[*c >> 4] + hex_chars[*c & 0xf];
                            ^
                    &       [                   ]
1 warning generated.
```

**Comment by mslehto** [comment]
Fixed by commit 7793cf157f7d8971db4ca3feb08900b4281c1802

**Comment by mslehto** [comment]
still fixed, closing
